inception-ui-core/src/main/java/de/tudarmstadt/ukp/inception/ui/core/dashboard/projectlist/ProjectsOverviewPage.properties

```diff
@@ -19,3 +19,6 @@ page.help=Help
 page.help.link=doc/user-guide.html#sect_menu
 
 newProject=Create new project ...
+
+LeaveDialog.title=Leave Project
```

Comment:
Naming convention is recommended as lowercase in property file. leaveDialog.title in this case is more appreciable.